espnet2/spk/projector/rawnet3_projector.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+from espnet2.spk.projector.abs_projector import AbsProjector
+
+
+class RawNet3Projector(AbsProjector):
+    def __init__(
+        self,
+        input_size,
+        output_size,
+    ):
+        super().__init__()
+        self._output_size = output_size
+
+        self.bn = torch.nn.BatchNorm1d(input_size)
+        self.fc = torch.nn.Linear(input_size, output_size)
+
+    def output_size(self):
+        return self._output_size
+
+    def forward(
+        self,
+        x,
+    ):
```

Comment:
```suggestion
    def forward(self, x):
```

Reply:
ditto

Reply:
I think this was done by precommit automatically

Reply:
No, if you remove the comma after `x`, it will actually give you the same code as I suggested.